cmd/kyma/kyma.go

```diff
@@ -64,7 +63,8 @@ For more information, see: https://github.com/kyma-project/cli
 		install.NewCmd(install.NewOptions(o)),
 		provisionCmd,
 		console.NewCmd(console.NewOptions(o)),
-		upgrade.NewCmd(upgrade.NewOptions(o)),
+		// upgrade command is disabled for now and will be enabled in the next release (1.16) after testing it on the upgrade prow jobs
```

Comment:
please dont put this as we should not commit features can cause unnecessary expectations